### PR TITLE
[Glimmer2] Migrate input tests

### PR DESCRIPTION
**Pull request by asakusuma**

### CODE REVIEW

**Comment by chadhietala** [comment]
@asakusuma Can you annotate the remove tests point back to the new tests so it's easier to confirm we are not loosing coverage?

**Comment by asakusuma** [comment]
@chadhietala what exactly do you mean by annotate? Can you give an example of this kind of annotation?

File: packages/ember-htmlbars/tests/helpers/input_test.js [hunk 1]
```diff
@@ -1,486 +0,0 @@
...
-    runAppend(view);
-  },
-
-  teardown() {
-    runDestroy(view);
-    runDestroy(owner);
-  }
-});
-
-QUnit.test('should insert a text field into DOM', function() {
```
Comment (chadhietala) [inline packages/ember-htmlbars/tests/helpers/input_test.js]: [Migrated](https://github.com/emberjs/ember.js/pull/13566/files#diff-0caed08a992497ff104284ec6aea6d96R44)

**Comment by chadhietala** [comment]
That ^

**Comment by GavinJoyce** [comment]
It seems like we've both done the same work: https://github.com/emberjs/ember.js/pull/13194

**Comment by asakusuma** [comment]
My bad @GavinJoyce. Closing in favor of https://github.com/emberjs/ember.js/pull/13194

**Pull request closed without merging**